scripts/verifier/diff-verifier-sequencer-state-dump.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Utility script for running a health check that compares the state of a verifier to the state of
+ * the current sequencer. Useful for catching discrepancies on verifier/sequencer codebases.
+ *
+ * Required environment variables:
+ * VERIFIER_ENDPOINT: RPC endpoint for the verifier node.
+ * SEQUENCER_ENDPOINT: RPC endpoint for the sequencer node.
+ */
+require("colors");
+const { Command } = require("commander");
+const axios = require("axios");
+const diff = require("deep-diff").diff;
+const fs = require("fs");
+const log = require("single-line-log").stdout;
+const dotenv = require("dotenv").config();
+const { JsonRpcProvider } = require("@ethersproject/providers");
+const program = new Command();
+
+program.option("-b, --block <number>", "specify block number");
+program.option("-s, --search", "binary search for specific diff");
+program.parse(process.argv);
+const argOptions = program.opts();
+
+const sources = [
+  {
+    url: `https://raw.githubusercontent.com/ethereum-optimism/regenesis/master/${process.env.ETH_NETWORK}/1.json`,
```

Comment:
might have to change this to 2.json actually
```suggestion
    url: `https://raw.githubusercontent.com/ethereum-optimism/regenesis/master/${process.env.ETH_NETWORK}/2.json`,
```